embulk-input-jdbc/src/main/java/org/embulk/input/jdbc/AbstractJdbcInputPlugin.java

```diff
@@ -66,10 +67,14 @@
         @ConfigDefault("null")
         public Optional<String> getOrderBy();
 
-        //// TODO See bellow.
-        //@Config("last_value")
-        //@ConfigDefault("null")
-        //public Optional<String> getLastValue();
+        @Config("incremental_column")
+        @ConfigDefault("null")
+        public Optional<List<String>> getIncrementalColumn();
```

Comment:
incremental_column option needs validation.

* all of incremental_columns must be included in the selected fields.
* if incremental, setting order_by is invalid. it should be rejected.